platform/install.go

```diff
@@ -0,0 +1,28 @@
+package platform
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	platformv1alpha1 "github.com/openshift/api/platform/v1alpha1"
+)
+
+const (
+	GroupName = "platform.openshift.io"
+)
+
+var (
+	schemeBuilder = runtime.NewSchemeBuilder(
+		platformv1alpha1.Install,
+	)
+	// Install is a function which adds every version of this group to a scheme
+	Install = schemeBuilder.AddToScheme
+)
+
+func Resource(resource string) schema.GroupResource {
+	return schema.GroupResource{Group: GroupName, Resource: resource}
+}
+
+func Kind(kind string) schema.GroupKind {
+	return schema.GroupKind{Group: GroupName, Kind: kind}
+}
```

Comment:
Were these copied from another package? Normally they are accompanied by some comments mentioning that they are deprecated

Reply:
Yes, I copied these from the `helm` package when I was creating it originally. Are you potentially thinking about the `register.go` files that have deprecation comments? A lot of `register.go` files with the deprecation comments but no `install.go`'s.

Reply:
I think it might be `register.go` I was thinking about yes